### ISSUE TITLE
Cannot build with Ubuntu on WSL 2

### ISSUE THREAD

**Issue by luisdelarosa** [ryanSN/winmoji#142]
**Describe the bug**
When I try to build using "yarn", I get the following info:
```
[3/5] Fetching packages...
info fsevents@1.2.9: The platform "linux" is incompatible with this module.
info "fsevents@1.2.9" is an optional dependency and failed compatibility check. Excluding it from installation.
```

Then when I execute "yarn dev", I get the following error:
```
/home/louie/projects/winmoji/node_modules/electron/dist/electron: error while loading shared libraries: libXcursor.so.1: cannot open shared object file: No such file or directory
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**To Reproduce**
```
yarn
yarn dev

```

**Expected behavior**
winmoji should build

**Desktop (please complete the following information):**
Microsoft Windows [Version 10.0.19592.1001]
Ubuntu 18.04.4 LTS on WSL 2
nvm
node v13.12.0

**Comment by ryanSN** [comment]
Would you be willing to try this suggestion? https://github.com/electron-userland/electron-prebuilt/issues/260

I personally use https://cmder.net/ on windows or the git-SCM
As you will need windows to build and deploy to windows (WSL won't work here).

**Comment by luisdelarosa** [comment]
Thanks! I'll try out both the electron workaround and also Cmder. I'm using to developing on OS X and Linux, so was trying to see if I could have a similar environment on Windows 10 with WSL. But I forgot that Electron natively targets Windows. :) Do I need anything else set up too like Visual Studio or anything specific? I'll also try installing yarn via Chocolatey.

**Comment by ryanSN** [comment]
Ah, yeah I develop on both.
So for windows check out Cmder, which uses git-scm to give you the linux commands etc..
I use https://github.com/coreybutler/nvm-windows for nvm like exp. 
I have yarn installed through Chocolatey. 

That should be enough to get going.

I've had nothing but trouble with WSL on Windows so I've given up until its a little more mature. 

Side note, this project also works on OS X 🤫 , I just only built it for Windows as thats what I was targeting at the time I made this.
